src/main/java/org/example/PolicyService.java

```diff
@@ -0,0 +1,68 @@
+package org.example;
+
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Scanner;
+
+public class PolicyService {
+
+    Repository repository;
+
+    public PolicyService(Repository repository) {
+        this.repository = repository;
+    }
```

Comment:
Really good separation here and passing in the Repository. This will now be easy to refactor if we need to switch the type of repository 👍

Reply:
👍